docs/ReferenceOneField.md

```diff
@@ -5,7 +5,7 @@ title: "The ReferenceOneField Component"
 
 # `<ReferenceOneField>`
 
-This field fetches a one-to-one relationship, e.g. the details of a book, when using a foreign key on the distant resource.
+This field fetches a one-to-one relationship or the first record of a one-to-many relationship, e.g. the details of a book, when using a foreign key on the distant resource.
```

Comment:
I would not mention this use-case so early in the doc, since it's after all a quite specialized use-case, and we want this first paragraph to introduce the most common use-case of the component only (let's keep things simple :wink: ).

Instead I'd mention this ability later, for instance as a **Tip**.

Reply:
Also, you need to update the `## Properties` section too, to document these new props